pkg/storage/driver/driver.go

```diff
@@ -39,44 +39,44 @@ type Creator interface {
 	Create(key string, rls *rspb.Release) error
 }
 
-// Updator is the interface that wraps the Update method.
+// Updater is the interface that wraps the Update method.
 //
 // Update updates an existing release or returns
 // ErrReleaseNotFound if the release does not exist.
-type Updator interface {
+type Updater interface {
 	Update(key string, rls *rspb.Release) error
 }
 
-// Deletor is the interface that wraps the Delete method.
+// Deleter is the interface that wraps the Delete method.
 //
 // Delete deletes the release named by key or returns
 // ErrReleaseNotFound if the release does not exist.
-type Deletor interface {
+type Deleter interface {
 	Delete(key string) (*rspb.Release, error)
 }
 
-// Queryor is the interface that wraps the Get and List methods.
+// Querier is the interface that wraps the Get and List methods.
```

Comment:
I half expect to get the most push-back (in this hunk) on this correction, but...